tests/framework/data/SqlDataProviderTest.php

```diff
@@ -32,7 +32,7 @@ public function testTotalCount()
             'sql' => 'select * from `customer`',
             'db' => $this->getConnection(),
         ]);
-        $this->assertSame(3, $dataProvider->getTotalCount());
+        $this->assertSame(3, (int) $dataProvider->getTotalCount());
```

Comment:
This is extremely confusing that `getTotalCount()` may return string in this case. :/